Drv/Ip/SocketReadTask.cpp

```diff
@@ -78,7 +79,7 @@ void SocketReadTask::readTask(void* pointer) {
                 "[WARNING] Failed to open port with status %d and errno %d\n",
                 static_cast<POINTER_CAST>(status),
                 static_cast<POINTER_CAST>(errno));
-            (void) Os::Task::delay(SOCKET_RETRY_INTERVAL_MS);
+            (void) Os::Task::delay(SOCKET_RETRY_INTERVAL);
```

Comment:
What is the reason for removing the units in the constant name? In general I think it's a good idea to include the units.

Reply:
OK, it looks like the type changed from integer to Fw::Time, and Fw::Time has its own units (seconds, microseconds).